README.md

```diff
@@ -20,6 +20,18 @@ npm install
 npm run dev
 ```
 
+## Setting up on cloud with Stackblitz Codeflow
+[Codeflow](https://developer.stackblitz.com/codeflow/what-is-codeflow) is a one-click integration with GitHub for seamless coding workflows.
+
+1. Initiate the setup process by clicking on the following button:
+
+[![Open in Codeflow](https://developer.stackblitz.com/img/open_in_codeflow.svg)](https:///pr.new/CircuitVerse/cv-frontend-vue)
```

Comment:
pr.new is used for pr reviews follow 
[https://developer.stackblitz.com/guides/integration/open-from-github](https://developer.stackblitz.com/guides/integration/open-from-github)
to setup the documentation

Reply:
thank you for the correction